sprokit/src/applets/pipeline_runner.cxx

```diff
@@ -105,25 +109,29 @@ run()
   }
 
   // Load the pipeline file.
-  kwiver::vital::path_t const pipe_file( cmd_args["pipe-file"].as<std::string>() );
+  kwiver::vital::path_t const pipe_file(
+    cmd_args[ "pipe-file" ].as< std::string >() );
   builder.load_pipeline( pipe_file );
 
   // Must be applied after pipe file is loaded.
   // To overwrite any existing settings
-  if ( cmd_args.count("config") > 0 )
+  if( cmd_args.count( "config" ) > 0 )
   {
-    std::vector< std::string > config_file_names = cmd_args["config"].as<std::vector<std::string>>();
-    for ( const auto& config : config_file_names )
+    std::vector< std::string > config_file_names =
+      cmd_args[ "config" ].as< std::vector< std::string > >();
+
+    for( const auto& config : config_file_names )
```

Comment:
BTW, optional, but since you're at it, you could change these to `auto const`...